tests/test_loaders_velociraptor.py

```diff
@@ -88,7 +88,7 @@ def test_velociraptor_loader_windows_ntfs(paths: List[str], mock_target: Target,
         (["uploads.json", "uploads/auto/Library", "uploads/auto/Applications"]),
     ],
 )
-def test_dir_loader_unix(paths: List[str], mock_target: Target, tmp_path: Path):
+def test_dir_loader_unix(paths: List[str], mock_target: Target, tmp_path: Path) -> None:
```

Comment:
```suggestion
def test_dir_loader_unix(paths: list[str], mock_target: Target, tmp_path: Path) -> None:
```
Some cleanup of old style type hints.